backend/testUtils/testSession.ts

```diff
@@ -1,22 +1,78 @@
 import {
-  ResultRequestDTO,
+  GradedResultResponseDTO,
+  UngradedResultDTO,
   TestSessionRequestDTO,
   TestSessionResponseDTO,
 } from "../services/interfaces/testSessionService";
+import { QuestionType } from "../models/test.model";
 
-export const testResult: ResultRequestDTO = {
+/**
+ * Mock Tests
+ */
+export const mockTest = {
+  id: "62c248c0f79d6c3c9ebbea95",
+  name: "test",
+  duration: 300,
+  admin: "62c248c0f79d6c3c9ebbea94",
+  questions: [
+    {
+      questionType: QuestionType.NUMERIC_ANSWER,
+      questionPrompt: "Numeric answer question",
+      questionMetadata: {
+        answer: 10,
+      },
+    },
+    {
+      questionType: QuestionType.MULTIPLE_CHOICE,
+      questionPrompt: "Multiple Choice question",
+      questionMetadata: {
+        options: ["11", "12", "13", "14"],
+        answerIndex: 0,
+      },
+    },
+    {
+      questionType: QuestionType.MULTIPLE_CHOICE,
+      questionPrompt: "Multiple Choice question",
+      questionMetadata: {
+        options: ["11", "12", "13", "14"],
+        answerIndex: 1,
+      },
+    },
+    {
+      questionType: QuestionType.NUMERIC_ANSWER,
+      questionPrompt: "Numeric answer question",
+      questionMetadata: {
+        answer: 14,
+      },
+    },
+  ],
+  grade: 11,
+};
+
+/**
+ * Mock Test Results
+ */
+export const mockUngradedTestResult: UngradedResultDTO = {
+  student: "some-student-name",
+  answers: [10, 11, 1, null],
```

Comment:
I've updated the result interfaces to `(number | null)[]` to allow us to store missing answers as null - lmk if this makes sense or if there's a better implementation / or if we are even allowing for blank answers in the first place? (can just make it required field in FE)